app/views/WhatDoYouWantToDoTileView.scala.html

```diff
@@ -41,22 +41,22 @@
   }
 }
 
+
 @mainTemplate(
     title = messages("your.paye.income.tax.overview"),
     authedUser = Some(user),
-    optimizelyAudience = optimizelyAudience,
```

Comment:
is it ok to remove this?

Reply:
there is no reference to   "optimizelyAudience" in the old maintemplate even though we are passing it in as a param so it looks like its unused.